### DISCUSSION TITLE
designable 是 MIT 协议吗？有朋友想一起fork出去继续维护和开源吗

### DISCUSSION THREAD

**Discussion by jinphic** [alibaba/designable#386] [Ideas]
因为这里暂停维护了....
希望作者大佬能回复下。

想参与“想fork一份出去继续维护和开源”的道友留个言～一起

**Comment by shijistar** [comment]
@jinphic https://github.com/pindjs/designable 这里已经有了不少开发了，是不是可以联系一下大家合力起来？貌似原作者已经不再维护了，这个项目放弃了很可惜
